test/e2e/host_path.go

```diff
@@ -77,27 +76,40 @@ var _ = Describe("hostPath", func() {
 	It("should support r/w", func() {
 		volumePath := "/test-volume"
 		filePath := path.Join(volumePath, "test-file")
+		retryDuration := 180
 		source := &api.HostPathVolumeSource{
 			Path: "/tmp",
 		}
 		pod := testPodWithHostVol(volumePath, source)
 
 		pod.Spec.Containers[0].Args = []string{
-			fmt.Sprintf("--fs_type=%v", volumePath),
 			fmt.Sprintf("--rw_new_file=%v", filePath),
 			fmt.Sprintf("--file_mode=%v", filePath),
 		}
+
 		testContainerOutputInNamespace("hostPath r/w", c, pod, 0, []string{
 			"mode of file \"/test-volume/test-file\": -rw-r--r--",
 			"content of file \"/test-volume/test-file\": mount-tester new file",
 		}, namespace.Name,
 		)
+
+		pod.Spec.Containers[1].Args = []string{
+			fmt.Sprintf("--file_content_in_loop=%v", filePath),
+			fmt.Sprintf("--retry_time=%d", retryDuration),
+		}
+		//Read the content of the file with the second container to
+		//verify volumes  being shared properly among continers within the pod.
+		testContainerOutputInNamespace("hostPath r/w", c, pod, 1, []string{
```

Comment:
Calling `testContainerOutputInNamespace` twice will run this pod twice -- is that what you intended to do?